Tools/Scripts/webkitdirs.pm

```diff
@@ -1284,9 +1324,6 @@ sub XcodeOptions
     push @options, "GCC_OPTIMIZATION_LEVEL=$forceOptimizationLevel" if $forceOptimizationLevel;
     push @options, "WK_LTO_MODE=$ltoMode" if $ltoMode;
     push @options, @baseProductDirOption;
-    push @options, "ARCHS=$architecture" if $architecture;
-    push @options, "ONLY_ACTIVE_ARCH=NO" if $didUserSpecifyArchitecture;
```

Comment:
Note to self, we should do something about `ONLY_ACTIVE_ARCH`. When passed, it invalidates incremental builds (because it changes script phase environment variables). And it's a no-op when building to a generic destination, since those have no concept of an "active" architecture.